index.html

```diff
@@ -12,7 +12,46 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
+    
+    <nav class="navbar">
+        <div class="container">
+            <img class="logo" src="img/karma-logo.svg" width="26">
+        <ul class="links">
+            <li>Meet karma</li>
+            <li>How it works</li>
+            <li>Store</li>
+            <li>Blog</li>
+            <li>Help</li>
+            <li>Login</li>
+        </ul>
+    
+    </nav>
+</div>
+    <div class="aside">
```

Comment:
Use a `main` tag to wrap everything that is not the header and the footer.

Reply:
The word `aside` implies that this is 'side' information, that it is nothing major. But you're using it here for the main hero section of the page. Choose a better adapted word that describes what it contains.